ffms2-sys/build.rs

```diff
@@ -16,11 +17,54 @@ fn format_write(builder: bindgen::Builder) -> String {
 }
 
 fn main() {
-    let libs = metadeps::probe().unwrap();
-    let headers = libs.get("ffms2").unwrap().include_paths.clone();
+    // Consider 'FFMS_INCLUDE_DIR' and 'FFMS_LIB_DIR', if pkg-config should not be used.
+    let headers = env::var("FFMS_INCLUDE_DIR").map(|value| {
+        // Ensure the include directory is valid
+        let include_dir = PathBuf::from(value.as_str());
```

Comment:
Can we use `Path` instead of `PathBuf`?

Reply:
The PathBuf is required to be compatible with the output of pkg-config

Reply:
Yeah, better to use `PathBuf` directly here than convert it later